core/src/com/sonyericsson/chkbugreport/plugins/battery/WakeLock.java

```diff
@@ -0,0 +1,116 @@
+package com.sonyericsson.chkbugreport.plugins.battery;
+
+import com.sonyericsson.chkbugreport.util.Util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WakeLock {
+    String mUID;
+    String mName;
+    String mType;
+    int mCount;
+    int mMax = -1;
+    int mActual = -1;
+    long mDurationMs;
+    Boolean mIsRunningLocked = false;
+    Boolean mIsRunning = false; //Only valid if mIsRunningLocked is true
+
+    private final Pattern pWLOuter = Pattern.compile("Wake lock (?:(\\S+) )?(\\S+)(:.*)? realtime");
+    private final Pattern pWLInner = Pattern.compile(": (.*?)(?: (\\D+) )?\\((\\d+) times\\)(?: max=(\\d+))?(?: actual=(\\d+))?(?: \\(running for (\\d+)ms\\))?( \\(running\\))?");
```

Comment:
Please update to use static definitons:
private static final Pattern WL_OUTER_PATTERN ...
private static final Pattern WL_INNER_PATTERN ...